ansible/roles/sonic-common/tasks/main.yml

```diff
@@ -18,6 +24,14 @@
 - include: aptrepo.yml
   tags: repo
 
+# Dbus
+- name: Install dbus
+  become: true
+  apt: pkg=dbus
```

Comment:
Tested on one switch (not AS7512). I don't think dbus is needed. Maybe you need 'sudo' before systemctl.
~$ sudo systemctl | head -2
  UNIT                                                                                     LOAD   ACTIVE SUB       DESCRIPTION
  proc-sys-fs-binfmt_misc.automount                                                        loaded active waiting   Arbitrary Executable File Formats File System Automount Point
~$ dpkg -l | grep dbus
~$